### PR TITLE
fluid_synth_nwrite_float(): write out effect buffers

### PR DESCRIPTION
**Pull request by derselbst**
finally implements proper multichannel rendering by also taking care of reverb and chrous channels

addresses #135

### CODE REVIEW

**Comment by derselbst** [comment]
@ensonic Please have a look at the API doc of fluid_synth_nwrite_float() and let me know, whether it's clear enough.

**Comment by derselbst** [comment]
@ensonic bump?

File: fluidsynth/src/synth/fluid_synth.c [hunk 1]
```diff
@@ -2501,6 +2559,25 @@ fluid_synth_nwrite_float(fluid_synth_t* synth, int len,
...
+#ifdef WITH_FLOAT
+      if(fx_left != NULL)
+        FLUID_MEMCPY(fx_left[i + count], fx_left_in[i], bytes);
+      
+      if(fx_right != NULL)
+        FLUID_MEMCPY(fx_right[i + count], fx_right_in[i], bytes);
+#else //WITH_FLOAT
+      int j;
+      for (j = 0; (fx_left != NULL) && (j < num); j++)
+          fx_left[i][j + count] = (float) fx_left_in[i][j];
```
Comment (ensonic) [inline fluidsynth/src/synth/fluid_synth.c]: I would not recheck this for each loop (also below)
``` C
if (fx_left != NULL)
{
  for (j = 0; j < num; j++)
     fx_left[i][j + count] = (float) fx_left_in[i][j];
} 
```

Comment (derselbst) [inline fluidsynth/src/synth/fluid_synth.c]: Oh ok, I usually trust todays compilers to generate something useful, but older compilers might have problems here, I agree.

**Pull request merged**